src/normalizer/mod.rs

```diff
@@ -84,14 +114,33 @@ where
             token_iter: Box::new(self),
             inner: Box::new(None.into_iter()),
             normalizer: NORMALIZERS.first().unwrap(),
+            options: None,
+        };
+        
+        NORMALIZERS.iter().skip(1).fold(first, |token_iter, normalizer| NormalizedTokenIter {
+            token_iter: Box::new(token_iter),
+            inner: Box::new(None.into_iter()),
+            normalizer,
+            options: None,
+        })
+    }
```

Comment:
we could just call `normalize_with_option` with `NormalizerOption::default()` instead of reimplementing the same thing.
Moreover, this would allow removing the `Option<NormalizerOption>` enum wrapper and directly having a `NormalizerOption`.